app/layout/project/navigation/index.tsx

```diff
@@ -55,19 +58,24 @@ export const Navigation = (): JSX.Element => {
   const isProjectRoute = route.startsWith('/projects/[pid]');
   const isScenarioRoute = route.startsWith('/projects/[pid]/scenarios/');
 
+  const { addToast } = useToasts();
+
   const [submenuState, setSubmenuState] = useState<{ [key in NavigationTreeCategories]: boolean }>({
     user: false,
     inventory: isProjectRoute && NAVIGATION_TREE.inventory.includes(tab),
     gridSetup: isScenarioRoute && NAVIGATION_TREE.gridSetup.includes(tab),
     solutions: isScenarioRoute && NAVIGATION_TREE.solutions.includes(tab),
     advancedSettings: isScenarioRoute && NAVIGATION_TREE.advancedSettings.includes(tab),
   });
+  const [scenarioRunning, setScenarioRunning] = useState(false);
```

Comment:
Can you please test if using the `status` property of the mutation helps to avoid using a state here? https://tanstack.com/query/v4/docs/react/reference/useMutation

Ideally, `status.isLoading` should do the same.